### PR TITLE
Nse 9400

### PR DESCRIPTION
**Pull request by visalachundru**
Please review my changes and let me know if there are any changes that are needed.

### CODE REVIEW

**Comment by covesturtevant** [comment]
@visalachundru Looks pretty good. A couple requested changes:

- Adjust pipe_list_li7200.txt to sort the pipeline specs in the order they need to be stood up. This file is used to programmatically stand up the pipelines in the correct order.
- merge in master and resolve all merge conflicts. Should just be the files you deleted.

**Comment by liangyingz** [comment]
@visalachundru 
I noted that only 3 days in output, can we run 12 days (01~12) instead? the downstream ECTE L1L4 need 9 days planar fit window.
In li7200_l0p_data.yaml, FILE_DIR needs to be data,flags, flags is missing
under pipe/l0_data_loader, the two li7200 related files are not needed anymore, you may also remove them

There are many resources adjustment, did you observe the server while pipelines running for those? (just curious)
@covesturtevant  I am always hesitate between SYMLINK and COPY, which one is better?

File: pipe/li7200/li7200_l0p_data.yaml [hunk 1]
```diff
@@ -3,20 +3,22 @@ pipeline:
...
   stdin:
   - "#!/bin/bash"
   - "python3 -m flow_sae_trst_dp0p.li7200"
   env:
     OUT_PATH: /pfs/out
-    FILE_DIR: data,flags
+    FILE_DIR: data
     LOG_LEVEL: INFO
-    RELATIVE_PATH_INDEX: "3"
+    NEW_SOURCE_TYPE_NAME: li7200
```
Comment (liangyingz) [inline pipe/li7200/li7200_l0p_data.yaml:15]: source type name not changed, this line is not needed

File: pipe/li7200/li7200_l0p_data.yaml [hunk 2]
```diff
@@ -3,20 +3,22 @@ pipeline:
...
-  image: quay.io/battelleecology/neon-sae-trst-dp0p-python:v0.3.2
+  image: quay.io/battelleecology/neon-sae-trst-dp0p-python:v0.4.1
   cmd: ["/bin/bash"]
   stdin:
   - "#!/bin/bash"
   - "python3 -m flow_sae_trst_dp0p.li7200"
   env:
     OUT_PATH: /pfs/out
-    FILE_DIR: data,flags
+    FILE_DIR: data
```
Comment (liangyingz) [inline pipe/li7200/li7200_l0p_data.yaml:13]: we need flags to include calibration flags

**Comment by covesturtevant** [comment]
> There are many resources adjustment, did you observe the server while pipelines running for those? (just curious) @covesturtevant I am always hesitate between SYMLINK and COPY, which one is better?

@liangyingz 
SYMLINK is generally better so long as 'empty_files' is not set to TRUE. Performance is terrible in that case.
COPY is required in the combined modules when writing to an interim directory because symbolic links cannot be daisy-chained. You'll get an error when the pipeline runs (or the pipeline will hang indefinitely) if this scenario arises.

**Comment by liangyingz** [comment]
@covesturtevant 
Thank you, this is very helpful!

**Pull request merged**